api/__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const supertest = require('supertest');
+const db = require("../../db/connection");
+const seed = require('../../db/seeds/seed');
+const app = require('../app');
+const data= require('../../db/data/test-data')
+
+beforeAll(() =>{
+    return seed(data)
+});
+
+
+describe.only('GET api/topics', () => {
+    test('should return a 200 status', () => {
+        return supertest(app).get('/api/topics').expect(200)
+    
+    })
+    test('should return an array of topic objects with correct properties', () => {
+        return supertest(app).get('/api/topics').expect(200)
+        .then(({body}) => {
+            expect(body.length).toBeGreaterThan(0);
```

Comment:
here it would be nice to check the actual length we want to get back rather than to just say greater than